data/images_to_samples_ci_csv.csv

```diff
@@ -1,2 +1,2 @@
-./data/22978945_15.tif,,./data/massachusetts_buildings.gpkg,,trn
-./data/23429155_15.tif,,./data/massachusetts_buildings.gpkg,,tst
+./22978945_15.tif,,./massachusetts_buildings.gpkg,,trn
+./23429155_15.tif,,./massachusetts_buildings.gpkg,,tst
```

Comment:
This is temporary. Will refactor the in_case_of_path parameter. I'd rather use hydra.utils.get_original_cwd().